SYCL/ESIMD/lsc/lsc_predicate.cpp

```diff
@@ -105,79 +107,80 @@ template <unsigned SIMDSize> int testUSM(queue q) {
   auto *vec_0 = malloc_shared<int>(size, q);
   auto *vec_1 = malloc_shared<int>(size, q);
   auto *vec_2 = malloc_shared<int>(size, q);
-  auto *vec_3 = malloc_shared<int>(size, q);
   std::iota(vec_0, vec_0 + size, 0);
   std::iota(vec_1, vec_1 + size, 0);
   std::iota(vec_2, vec_2 + size, 0);
-  std::iota(vec_3, vec_3 + size, 0);
 
   try {
     q.submit([&](handler &h) {
       h.parallel_for(
           range<1>{size / SIMDSize}, [=](id<1> id) SYCL_ESIMD_KERNEL {
             auto offset = id[0] * SIMDSize;
-            auto offsets = simd<uint32_t, SIMDSize>(id * SIMDSize * sizeof(int),
-                                                    sizeof(int));
+
             auto pred_enable = simd_mask<1>(1);
             auto pred_disable = simd_mask<1>(0);
 
             auto data_0 =
                 lsc_block_load<int, SIMDSize>(vec_0 + offset, pred_enable);
             lsc_block_store<int, SIMDSize>(vec_0 + offset, data_0 * 2,
                                            pred_enable);
-
             auto data_1 =
-                lsc_block_load<int, SIMDSize>(vec_1 + offset, pred_disable);
+                lsc_block_load<int, SIMDSize>(vec_1 + offset, pred_enable);
             lsc_block_store<int, SIMDSize>(vec_1 + offset, data_1 * 2,
-                                           pred_enable);
-
+                                           pred_disable);
             auto data_2 =
-                lsc_block_load<int, SIMDSize>(vec_2 + offset, pred_enable);
+                lsc_block_load<int, SIMDSize>(vec_2 + offset, pred_disable);
             lsc_block_store<int, SIMDSize>(vec_2 + offset, data_2 * 2,
                                            pred_disable);
```

Comment:
This test case is just redundant, right?
If lsc_block_load with pred_diable works incorrectly, the test only verifies lsc_block_store on the line 133, and such check already exists on the line 129.